dashboard/Home.py

```diff
@@ -0,0 +1,35 @@
+import streamlit as st
+
+st.set_page_config(
+    page_title="RailWatch",
+    page_icon="🚆",
+    layout="wide",
+    initial_sidebar_state="expanded",
+)
+
+with st.sidebar:
+    st.image("logo.png", use_column_width=True)
+
+
+# Page title
+st.title("RailWatch: The Train Station Tracker")
+
+# Main section
+st.header("Welcome!")
+
+st.markdown(
+    """
```

Comment:
Are we moving it to another page?

Reply:
I'll make another pull request for this down the line.